WordPress/Classes/ViewRelated/Stats/Period Stats/SiteStatsPeriodViewModel.swift

```diff
@@ -228,10 +228,15 @@ private extension SiteStatsPeriodViewModel {
     }
 
     func countriesDataRows() -> [StatsTotalRowData] {
-        return store.getTopCountries()?.map { StatsTotalRowData.init(name: $0.label,
-                                                                     data: $0.value.displayString(),
-                                                                     countryIconURL: $0.iconURL,
-                                                                     statSection: .periodCountries) }
+        return store.getTopCountries()?.map {
+            // This hack is here just temporarily until #11342 gets merged which exposes the "code" property nicely.
```

Comment:
It appears that #11342 has been merged. Can this "hack" be removed now?

Reply:
Fixed!